warp/handlers/signature_request_test.go

```diff
@@ -64,27 +61,28 @@ func TestSignatureHandler(t *testing.T) {
 					MessageID: unknownMessageID,
 				}, emptySignature[:]
 			},
-			verifyStats: func(t *testing.T, stats *stats.MockSignatureRequestHandlerStats) {
-				require.EqualValues(t, 1, mockHandlerStats.SignatureRequestCount)
-				require.EqualValues(t, 1, mockHandlerStats.SignatureRequestMiss)
-				require.EqualValues(t, 0, mockHandlerStats.SignatureRequestHit)
-				require.Greater(t, mockHandlerStats.SignatureRequestDuration, time.Duration(0))
+			verifyStats: func(t *testing.T, stats *handlerStats) {
+				require.EqualValues(t, 1, stats.signatureRequest.Count())
+				require.EqualValues(t, 0, stats.signatureHit.Count())
+				require.EqualValues(t, 1, stats.signatureMiss.Count())
+				require.Greater(t, stats.signatureProcessingTime.Value(), time.Duration(0))
 			},
 		},
 	}
 
 	for name, test := range tests {
-		// Reset stats before each test
-		mockHandlerStats.Reset()
-
 		t.Run(name, func(t *testing.T) {
+			handler := NewSignatureRequestHandler(backend, message.Codec)
+			handler.stats.Clear()
```

Comment:
isn't this already clean?

Reply:
No, these are globally registered, so we need to clear after each run.